### PR TITLE
Fixes ambiguous inheritdocs

### PR DESCRIPTION
**Pull request by distributivgesetz**
[Base Branch]: master

[Release Notes]: 

Nothing user-facing

[Why]: make it so my IDE yells at me less loudly

### CODE REVIEW

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Changelog is only for user-facing changes so could you remove it please?

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Also can you scope these appropriately so they don't trigger `IDE0001` on VS please?
image

**Comment by distributivgesetz** [comment]
this is gonna take some adjusting lol

File: src/Tgstation.Server.Host/IO/Console.cs [hunk 1]
```diff
@@ -7,7 +7,10 @@
 
 namespace Tgstation.Server.Host.IO
 {
-	/// <inheritdoc />
+	/// <summary>
+	/// Implementation of <see cref="Tgstation.Server.Host.IO.IConsole"/>,
```
Comment (Cyberboss) [inline src/Tgstation.Server.Host/IO/Console.cs:11]: Missed this one

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: src/Tgstation.Server.Host/IO/Console.cs (hunk 1 above)
Comment (distributivgesetz) [reply src/Tgstation.Server.Host/IO/Console.cs:11]: I'm so close to pulling the trigger on Rider I swear to god

**Review by Cyberboss — APPROVED** [review]

**Comment by Cyberboss** [comment]
Docker build is broken because of the node version we were using getting too old. #1647 fixes it. Can you merge master once that goes through?

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**